### PR TITLE
Meta images sanity check

### PR DESCRIPTION
**Pull request by michalsidzej**
Resolves L2B-294
In the end, I decided to check if a meta image has the L2beat logo in it. This will give us the most confidence that the image is right.

### CODE REVIEW

File: packages/frontend/src/build/buildMetaImages.ts [hunk 1]
```diff
@@ -4,12 +4,18 @@ import { mkdirpSync } from 'fs-extra'
...
 
+import { getConfig } from './config'
+
 main().catch((e) => {
   console.error(e)
   process.exit(1)
 })
 
 async function main() {
+  const env = process.env.DEPLOYMENT_ENV ?? 'production'
```
Comment (krzkaczor) [inline packages/frontend/src/build/buildMetaImages.ts:15]: we already have this in the buildPages, do we need it here too?

File: packages/frontend/src/build/buildMetaImages.ts [hunk 2]
```diff
@@ -47,3 +60,11 @@ async function main() {
     server.close((err) => (err ? reject(err) : resolve())),
   )
 }
+
+async function sanityCheck(page: puppeteer.Page, slug: string) {
```
Comment (krzkaczor) [inline packages/frontend/src/build/buildMetaImages.ts:64]: please, provide explcit return type here.

File: packages/frontend/src/build/buildMetaImages.ts [hunk 3]
```diff
@@ -47,3 +60,11 @@ async function main() {
...
+  const l2BeatLogo = await page.$('[aria-label="L2BEAT logo"]')
+  if (!l2BeatLogo) {
+    const string = `Meta image for ${slug} did not build properly!`
+    throw new Error(string)
```
Comment (krzkaczor) [inline packages/frontend/src/build/buildMetaImages.ts:68]: is this "string" var really better than inlining it? ;d

File: packages/frontend/src/build/buildMetaImages.ts (hunk 1 above)
Comment (michalsidzej) [reply packages/frontend/src/build/buildMetaImages.ts:15]: `buildPages` runs in a separate process, so yes, I think we need it here too.

**Review by krzkaczor — APPROVED** [review]

**Pull request merged**